src/react-apps/applications/ux-editor/src/actions/formDesignerActions/actions/updateApplicationMetadata/index.ts

```diff
@@ -0,0 +1,53 @@
+import { Action } from 'redux';
+import * as ActionTypes from '../../formDesignerActionTypes';
+
+export interface IUpdateApplicationMetadaAction extends Action {
+  id: string;
+  maxFiles: number;
+  maxSize: number;
+  fileType: string;
+}
+
+export interface IUpdateApplicationMetadaActionFulfilled extends Action {
```

Comment:
All these props used for anything? If not remove

Reply:
Fixed